app/Filament/Resources/ApiKeyResource/Pages/ListApiKeys.php

```diff
@@ -51,13 +52,23 @@ public function table(Table $table): Table
             ])
             ->actions([
                 DeleteAction::make(),
+            ])
+            ->emptyStateIcon('tabler-key')
+            ->emptyStateDescription('')
+            ->emptyStateHeading('No API Keys')
+            ->emptyStateActions([
+                CreateAction::make('create')
+                    ->label('Create API Key')
+                    ->button(),
             ]);
     }
 
     protected function getHeaderActions(): array
     {
         return [
-            Actions\CreateAction::make(),
+            Actions\CreateAction::make()
+                ->label('New API Key')
+                ->hidden(fn () => ApiKey::count() <= 0),
```

Comment:
Why would you hide it if there's no key yet ?

Reply:
It's shown in the empty state